api/src/main/java/uk/gov/dhsc/htbhf/claimant/message/MoneyUtils.java

```diff
@@ -0,0 +1,30 @@
+package uk.gov.dhsc.htbhf.claimant.message;
+
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+import java.text.NumberFormat;
+import java.util.Locale;
+
+/**
+ * Contains utility methods for dealing with monetary amounts.
+ */
+public class MoneyUtils {
+
+    private static final int SCALE_2_DP = 2;
```

Comment:
`NUM_DECIMAL_PLACES`

Reply:
Updated implementation doesn't need this constant. :-)